### PR TITLE
PR 5098+ 4984 issue #5072

### PR DESCRIPTION
**Pull request by j0rt3g4**
PR #5098 + PR #4984 
for issue #5072

### CODE REVIEW

**Comment by j0rt3g4** [comment]
@illfated  I've added all both

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by illfated** [comment]
For easier overview, Rich Text Diff:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5328/files?short_path=46e8921#diff-46e8921ff4540914a5b23145a04ec20b

File: devices/surface/surface-dock-firmware-update.md [hunk 1]
```diff
@@ -84,8 +90,8 @@ Successful completion of Surface Dock Firmware Update results in new registry ke
...
-| Surface Dock Firmware Update log | /l*v %windir%\logs\Applications\SurfaceDockFWI.log | Earlier versions of this tool wrote events to Applications and Services Logs\Microsoft Surface Dock Updater.                                                                                                  |
-| Windows Device Install log       | %windir%\inf\ setupapi.dev.log         | For more information about using Device Install Log, refer [to SetupAPI Logging](https://docs.microsoft.com/windows-hardware/drivers/install/setupapi-logging--windows-vista-and-later-) documentation. |
+| Surface Dock Firmware Update log | Path needs to be specified (see note) | Earlier versions of this tool wrote events to Applications and Services Logs\Microsoft Surface Dock Updater.                                                                                                  |
+| Windows Device Install log       | %windir%\inf\setupapi.dev.log         | For more information about using Device Install Log, refer [to SetupAPI Logging](https://docs.microsoft.com/windows-hardware/drivers/install/setupapi-logging--windows-vista-and-later-) documentation. |
```
Comment (JohanFreelancer9) [inline devices/surface/surface-dock-firmware-update.md]:
```suggestion
| Windows Device Install log       | %windir%\inf\setupapi.dev.log         | For more information about using Device Install Log, refer to [SetupAPI Logging](https://docs.microsoft.com/windows-hardware/drivers/install/setupapi-logging--windows-vista-and-later-). |
```

Comment (mypil) [reply devices/surface/surface-dock-firmware-update.md]: @j0rt3g4  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by j0rt3g4** [comment]
@mypil  done.
Thank you @JohanFreelancer9  as usual!.

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #5072. 

Thank you.

cc: @kenwith

**Pull request merged**